keras/layers/advanced_activations.py

```diff
@@ -202,7 +202,7 @@ def __init__(self, theta=1.0, **kwargs):
         self.theta = K.cast_to_floatx(theta)
 
     def call(self, inputs, mask=None):
-        return inputs * K.cast(inputs > self.theta, K.floatx())
+        return inputs * K.cast(K.greater(inputs, self.theta), K.floatx())
```

Comment:
In principle, we can override `>`, `<`, `>=`, and `<=`. The only one we cannot really override is == which cntk has reserved for C++-side object equality so we can write things like 
```python
for tensor in list_of_tensors:
...
```
This is the reason why we have not introduced infix comparison operators. The risk is someone will use == to mean cntk.equal leading to unexpected behavior, while the benefit is syntactic sugar. For now, we have opted to avoid these surprises.